src/triage/predictlist/utils.py

```diff
@@ -0,0 +1,150 @@
+from triage.component.results_schema import RetrainModel, Retrain
+from triage.component.catwalk.utils import db_retry, filename_friendly_hash
+
+import re
+from sqlalchemy.orm import sessionmaker
+import verboselogs, logging
+logger = verboselogs.VerboseLogger(__name__)
+
+
+def experiment_config_from_model_id(db_engine, model_id):
+    """Get original experiment config from model_id 
+    Args:
+            db_engine (sqlalchemy.db.engine)
+            model_id (int) The id of a given model in the database
+
+    Returns: (dict) experiment config
+    """
+    get_experiment_query = '''select experiments.config
+    from triage_metadata.triage_runs
+    join triage_metadata.models on (triage_runs.id = models.built_in_triage_run)
+    join triage_metadata.experiments on (experiments.experiment_hash = triage_runs.run_hash)
```

Comment:
probably shouldn't expect collisions, but it might be good to be explicit here, e.g.
```
join triage_metadata.experiments on (experiments.experiment_hash = triage_runs.run_hash and triage_runs.run_type='experiment')
```